dist/scss/variables.scss

```diff
@@ -111,6 +111,6 @@ $navigationHoverColor: #191D22;
 // Overlay
 // -------------------------
 $overlayContainerWidthSmall: 580px;
-$overlayContainerWidthMedium: 700px;
+$overlayContainerWidthMedium: 860px;
```

Comment:
Why is that necessary? There is almost no difference anymore between medium and large...